src/pages/task/taskTabs/logs/logTypes/TaskEventLogLine.tsx

```diff
@@ -17,6 +17,9 @@ export const TaskEventLogLine: React.FC<TaskEventLogEntry> = ({
   const containerOrHostCopy = podId ? "container" : "host";
   let message: JSX.Element;
   switch (eventType) {
+    case TaskEventType.TaskBlocked:
+      message = <>This task is blocked.</>;
```

Comment:
nit: the tense of this one is a little different from the other ones already here, should it say something like `Task is blocked` or `Task blocked` for consistency with the others?